Messerli.ChangeCase.Test/FirstCharacterToLowerCaseTest.cs

```diff
@@ -12,6 +12,10 @@ public class FirstCharacterToLowerCaseTest
         [InlineData("this_is_a_long_snake_case_name", "this_is_a_long_snake_case_name")]
         [InlineData("HTML", "hTML")]
         [InlineData("htmlDocument", "htmlDocument")]
+        [InlineData("end_2_end", "end_2_end")]
+        [InlineData("BUSINESS_2_BUSINESS", "bUSINESS_2_BUSINESS")]
+        [InlineData("End2End", "end2End")]
+        [InlineData("business2Business", "business2Business")]
```

Comment:
There are no tests for numbers that consist of more than one digit. Is that intentional?

Reply:
Added and implemented in b6a105f61574ed19e5a2af14554013d4a6a941da